scripts/install-okctl.sh

```diff
@@ -86,6 +92,9 @@ check_requirements() {
 install_binary() {
     local version=$1
     local BINARY_NAME="okctl"
+    if [ "$PROXY" == "CN" ]; then
```

Comment:
I think it's not necessary to add CN here, if --proxy is passed then the user is meant to use the proxy address

Reply:
OK, I've modified it.